ortho_map.txt

```diff
@@ -82,8 +69,24 @@ Layer 4
 ................................
 ................................
 ................................
-...................F11111.......
-....................F1111.......
+
+LAYER 4
+...................F11111J......
+....................F11111J.....
 .....................F111.......
 ......................F11.......
-.......................F1.......
\ No newline at end of file
+.......................F1.......
+....1...........................
+................................
+................................
+................................
+................................
+................................
+................................
+................................
+................................
+................................
+................................
+
+DATA
+Player Start: 7.5, 1.5, 1
```

Comment:
We can take time later to figure out some conventions about how to organize our data in the map file. For now, it's important to make sure the section labels (e.g. 'LAYER 0' or 'DATA') match the part of the `MapClass` code that parses this file. Also, as the parsing code exists currently, all layer data needs to go at the beginning; anything else needs to go below the layers.